package.json

```diff
@@ -37,7 +37,8 @@
     "mustache": "^2.3.0",
     "optipng-bin": "^4.0.0",
     "pngquant": "^1.3.0",
-    "pngquant-bin": "^3.1.1"
+    "pngquant-bin": "^3.1.1",
+    "uglifyjs-webpack-plugin": "^0.4.6"
```

Comment:
uglifyjs-webpack-plugin is already part of devDependencies so we don't need it here. Also handled in #15, so maybe just revert this change.